lib/screens/authentication/recover/recover_account_found/recover_account_found_errors.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+enum RecoverAccountFoundError{ NoGuardians, Unknown}
```

Comment:
@gguijarro-c-chwy 
We must standarize using low case for enum values.

Reply:
yeah, i see dart has them as low case 

Using enums
Declare an enumerated type using the enum keyword:

enum Color { red, green, blue }

is this something we can catch with linter?

Reply:
Yes we can catch them by a lint 
https://dart-lang.github.io/linter/lints/constant_identifier_names.html

Reply:
> We must standarize using low case for enum values.

Ah, yes, good catch, @RaulUrtecho!

> In new code, use lowerCamelCase for constant variables, including enum values.

https://dart.dev/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names

Reply:
@RaulUrtecho  pls activate this lint

Reply:
I'll do in a new PR